### PR TITLE
Make `import()` options actually optional

### PR DESCRIPTION
**Pull request by mattvague**
As it sounds. Currently you have to pass an empty object to make TS happy

image

### CODE REVIEW

**Comment by gwhitney** [comment]
Please add a test to the math.import section of types/index.ts that exercises the improved declaration of import, thanks. Otherwise this PR is clearly good to go.

**Comment by mattvague** [comment]
>Please add a test to the math.import section of types/index.ts that exercises the improved declaration of import, thanks. Otherwise this PR is clearly good to go.

Whoa! That is super cool, I've never seen type tests like that before -- makes a lot of sense for a library. Done!

**Comment by mattvague** [comment]
I was also automatically added to the `AUTHORS.md` list, should I leave that?

**Comment by gwhitney** [comment]
Yes, once this is merged you should indeed be on the AUTHORS list, so fine to leave that. And I know some projects that would balk at adding tests for .toTex in a PR that has to do with .import, but I don't mind and I don't think Jos does either. However, index/types.ts now has two sections that deal with math.import, which is confusing, so please coalesce them (into whatever location in the file you feel is most appropriate). Also, I think it is the convention for mathjs source files to end with a newline, so please make sure index/types.ts does. Thanks for polishing this up!

**Comment by mattvague** [comment]
Done, done and done

**Comment by mattvague** [comment]
@gwhitney Whoops, sorry about that, guess I forgot to run the tests before I pushed. Should be good to go now

File: types/index.ts [hunk 1]
```diff
@@ -756,36 +756,36 @@ declare module 'mathjs' {
...
+    myvalue1: 42,
+    myFunc: (name: string) => `myFunc ${name}`
   })).toMatchTypeOf<void>()
 
   expectTypeOf(math.import({
-    myvalue: 42
+    myvalue2: 42
   }, {
     override: true
   })).toMatchTypeOf<void>()
```
Comment (gwhitney) [inline types/index.ts:768]: Why was it necessary to change to `myvalue2` here? Doesn't the `override` option make it OK to redefine an existing symbol?

Comment (mattvague) [reply types/index.ts:768]: Oh, no that probably wasn't necessary. Want me to change?

Comment (gwhitney) [reply types/index.ts:768]: I mean, leaving it as myvalue when all the other tests use a new name does sort of highlight/test the meaning of override...

Comment (mattvague) [reply types/index.ts:768]: True. Will fix

**Comment by mattvague** [comment]
@gwhitney Hate to bug ya but any chance we can get this merged if it’s good to go?

**Comment by josdejong** [comment]
Looks good @mattvague , I'll merge the PR now. Thanks!

**Comment by josdejong** [comment]
There where some merge conflicts due to an other PR I just merged, I fixed those conflicts and merged it manually.

**Pull request merged**